poetry_to_pre_commit/common.py

```diff
@@ -35,6 +59,5 @@ def get_poetry_packages(cwd: pathlib.Path | None = None) -> Iterable[PoetryPacka
 
     for package in repository.packages:
         yield PoetryPackage(
-            name=package.name,
-            version=package.version.text,
+            name=package.name, version=package.version.text, extras=package.features
         )
```

Comment:
I am actually wondering why we need our own `PoetryPackage`. Couldn't we directly return `repository.packages` and work with instances of `poetry.core.packages.package.Package`?

Reply:
It's not something we _have_ to do, it would work well that way too but it's far from being a bad idea to try and isolate third parties.
Here, the codebase is really small, so it doesn't matter a lot, but I don't think it's better to pass Poetry's objects to our own functions more than necessary.

Reply:
Fair enough.